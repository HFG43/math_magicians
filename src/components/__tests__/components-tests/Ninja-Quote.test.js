import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../ninja';

it('Renders <Quote /> correctly', () => {
  const quote = renderer.create(<Quote />).toJSON();
  expect(quote).toMatchSnapshot();
});
