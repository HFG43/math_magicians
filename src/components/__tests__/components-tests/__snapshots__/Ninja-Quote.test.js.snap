// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders <Quote /> correctly 1`] = `
<div>
  There info is loading... Please wait!
</div>
`;
